app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -107,9 +107,29 @@ def textual_endpoints
       :value => @record.connection_configurations.hawkular.endpoint.port}]
   end
 
+  def redact_passwords_from_proxy_values(name, value)
+    unless ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::PROXY_ENV_VARIABLES.include?(name)
```

Comment:
Ideally, things like model names (`ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job:`)
should not show in UI code and the above is a pattern we'd like to get rid of.

In this case, there could be a general mechanism where UI would be asking the provider object which
fields should be redacted.